mstyle/mstyle.cpp

```diff
@@ -1290,8 +1290,8 @@ bool MgStyle::drawFrameGroupBoxPrimitive(const QStyleOption* option, QPainter* p
             return true;
 
       // no frame for flat groupboxes
-      QStyleOptionFrameV2 fOpt2(*fOpt);
-      if (fOpt2.features & QStyleOptionFrameV2::Flat)
+      QStyleOptionFrame fOpt2(*fOpt);
+      if (fOpt2.features & QStyleOptionFrame::Flat)
```

Comment:
Seems it was only introduced in 5.5
http://doc.qt.io/qt-5/qframe.html#initStyleOption